packages/shared/src/modules/users/libs/enums/user-validation-message.enum.ts

```diff
@@ -27,7 +26,8 @@ const UserValidationMessage = {
 	PASSWORD_MIN_LENGTH: `Password must be at least ${String(
 		UserValidationRule.PASSWORD_MIN_LENGTH,
 	)} characters long`,
-	PASSWORD_REQUIRE: "Password is not allowed to be empty",
+	PASSWORD_NOT_MATCH: "Passwords do not match. Please re-enter your password",
```

Comment:
We don't need this message here, because ConfirmPasswordCustomValidation enum already holds this message. Please, update error message in packages/shared/src/modules/auth/libs/enums/confirm-password-custom-validation.enum.ts.